src/app/view/components/dot-navigation/components/dot-nav-item/dot-nav-item.component.scss

```diff
@@ -5,6 +5,29 @@
 :host {
     display: block;
 
+    &:not(.collapsed) {
+        ::ng-deep {
+            .dot-nav__tooltip {
+                display: none !important;
+            }
+        }
+    }
+
+    ::ng-deep {
+        .ui-tooltip {
+            &.dot-nav__tooltip {
+                margin-left: $basic-padding + $navigation-width-collased - $navigation-width;
```

Comment:
Isn't this dynamically set by the tooltip directive?

Reply:
no with the current implementation, because part of the target element is hidden. maybe we need to change the current approach ?